python/pfs/drp/stella/reduceArcTask.py

```diff
@@ -264,7 +261,7 @@ def run(self, expRefList, butler, wLenFile=None, lineList=None, immediate=True):
             else:
                 print 'setSpectrum for spectrumSetFromProfile[',i,'] failed'
 
-        if True:
+        if False:
```

Comment:
Hmm. LSST's [RFC-205](https://jira.lsstcorp.org/browse/RFC-205) for a discussion of this. It looks like they're converging on a decision to discourage this idiom, and we've agreed to follow LSST standards... but they haven't quite converged yet, so maybe you can get away with it for now. However, I definitely think we'd require a comment explaining what's going on here.